packages/runtime/src/helpers/flags.ts

```diff
@@ -29,3 +29,10 @@ export const splitApiRoutes = (featureFlags: Record<string, unknown>, publish: s
 
   return isEnabled
 }
+
+export const bundleBasedOnNftFiles = (featureFlags: Record<string, unknown>): boolean => {
```

Comment:
Note to self: We should probably make `splitApiRoute` another activation guard for this.